docs/developer-docs/latest/developer-resources/content-api/content-api.md

```diff
@@ -1095,3 +1101,7 @@ OR
 If you only want to retrieve your draft entries, you can combine the `preview` mode and the `published_at` field.
 `GET /articles?_publicationState=preview&published_at_null=true`
 :::
+
+### Locale
+
+If the [Internationalization (i18n) plugin](/developer-docs/latest/development/plugins/i18n.md) is installed, and the [appropriate permission has been set](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md#collection-and-single-types) for the content-type, the `locale` API parameter can be used to [get entries from a specific locale](/developer-docs/latest/development/plugins/i18n.md#getting-localized-entries-with-the-locale-parameter).
```

Comment:
I actually wasn't refering to permissions :) 
Once i18n is installed it is not enabled on the content-types you need to enable it per content type before having the new queries :)

Reply:
I seeeee! Thanks Alex, I missed this step because I used a pre-configured app 😅